test/fixtures/output/connect/custom-plugin/version/create-help.golden

```diff
@@ -0,0 +1,24 @@
+Create a custom connector plugin version.
+
+Usage:
+  confluent connect custom-plugin version create [flags]
+
+Examples:
+Create custom connector plugin version for plugin "ccp-123456".
+
+  $ confluent connect custom-plugin version create --plugin ccp-123456 --plugin-file datagen.zip --version-number 0.0.1 --beta=false --sensitive-properties=property1,property2
```

Comment:
Are "property1,property2" actual property keys that we would see in production?

Reply:
Maybe `aws.key,aws.value` would be better here?

Reply:
it could be keys, tokens. let me modify as what we have for api doc:
```
"sensitive_config_properties": [
"passwords",
"keys",
"tokens"
],
```